float64 distance_front
int32 angle_front
float64 distance_back
int32 angle_back
float64 turn_left
string turn_left_sensor
float64 turn_right
string turn_right_sensor

